lighthouse-core/lib/tracehouse/trace-processor.js

```diff
@@ -604,6 +604,13 @@ class TraceProcessor {
         return Boolean(data && data.frame && data.url);
       });
 
+    const layoutShiftTimelineEvents = keyEvents
```

Comment:
we would probably move this out into the audit/gatherer that needs it since it's pretty specific